ext/rotoscope/rotoscope.h

```diff
@@ -3,6 +3,8 @@
 
 #include "unistd.h"
 
+#include "callsite.h"
```

Comment:
If we don't need any of the types for `rotoscope.h` does it make sense to move this include into `rotoscope.c`? That's what I've done with the other includes but I don't know what's more standard (I'm aware they're functionally equivalent though)